spec/spec_helper.rb

```diff
@@ -22,13 +22,7 @@
   when 'darwin'
     'launchd'
   when 'debian'
-    if facts[:operatingsystem] == 'Ubuntu'
```

Comment:
I'm slightly tempted to keep this in since it matches the Puppet built in values and externalize this somewhere, but probably fine to drop it here.

Reply:
we need to handle this in msync later anyway.